### DISCUSSION TITLE
❓ Pipe key does nothing

### DISCUSSION THREAD

**Discussion by vamfoom** [isontheline/pro.webssh.net#329] [Q&A]
I use WebSSH on an iPad pro with the Apple Magic Keyboard.  The pipe ( | ) key does nothing when ssh’ing to Linux hosts.  Is there a fix coming for this?

**Comment by isontheline** [comment]
Hello @vamfoom 👋

Does the pipe key works inside a textfield when editing a connection?

Have a nice day ☀️

**Reply by isontheline** [reply]
@vamfoom Any news about my question please?

**Reply by isontheline** [reply]
@vamfoom May I close this discussion?
